src/main/haskell/kore/src/Kore/Builtin/Set.hs

```diff
@@ -370,61 +402,91 @@ asExpandedPattern symbols resultSort =
     asExpandedPattern0 = \asPattern0 builtin ->
         ExpandedPattern.fromPurePattern $ asPattern0 builtin
 
+concatKey :: String
+concatKey = "SET.concat"
```

Comment:
Could you use these constants in other places where their value is used, e.g. Test.Kore.Builtin.Definition?

Reply:
I'll add the `Set` ones now and create a different PR to update all builtins in the same way.